### PR TITLE
Adds search by name, type, cuisine

### PR DESCRIPTION
**Pull request by ivan-1kh**
rel #8

### CODE REVIEW

File: src/Languages/ar.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ export default {
     'signin': 'تسجيل الدخول',
     'signup': ' مستخدم جديد',
     'topRated': 'اعلى التقيمات',
+    'searchResults': 'سيرتش ريزولتس',
```
Comment (Karyum) [inline src/Languages/ar.js]: really?

Comment (ivan-1kh) [reply src/Languages/ar.js]: Put it temporarily, forgotten to fix it, whoopsie.
It's نتائج البحث now

**Review by Karyum — COMMENTED** [review]
what do you think about this @ivankhuri 
we have the search bar on top of the map (specifically top middle of the map) and when someone tries to filter then it would filter the markers on the map

**Comment by ivan-1kh** [comment]
> what do you think about this @ivankhuri
> we have the search bar on top of the map (specifically top middle of the map) and when someone tries to filter then it would filter the markers on the map

I will do this, but how about the search applying both to the slider and the map?

**Comment by Karyum** [comment]
I imagine later on the slider would be the `promoted businesses` , or maybe a businesses who paid to get on the front page. so no need to filter it

**Comment by ivan-1kh** [comment]
> I imagine later on the slider would be the `promoted businesses` , or maybe a businesses who paid to get on the front page. so no need to filter it

So only map?

**Comment by Karyum** [comment]
yes

**Comment by ivan-1kh** [comment]
> yes

done

**Review by Karyum — APPROVED** [review]

**Pull request merged**